### ISSUE TITLE
\refnp failing in Bible Modules

### ISSUE THREAD

**Issue by mnjames** [sillsdev/ptx2pdf#720]
When using the Bible Modules option to print in ptxprint, sometimes a reference which is given using the \refnp code is created as a new paragraph and not included as part of the previous one. I've spent an embarrassing amount of time trying to figure out which conditions trigger it and can't (it often works fine).

------------------------

Also, does ptxprint cache files? Recently it seems like if I make changes in PT and then print a Bible Module, the changes don't show up unless I shut down ptxprint and restart it.

**Comment by mhosken** [comment]
Please don't raise two bugs in one issue. I've made fixes, but with no project I can't test. Please pull and test or wait for the next release (probably later today).

**Comment by mnjames** [comment]
Using 2.2.19, I get the following error when trying to print Bible Modules via both the Single Book (XXA) option or via the Bible Module option.
````
Traceback (most recent call last):
  File "ptxprint\gtkview.py", line 1253, in onOK
  File "ptxprint", line 223, in doit
  File "ptxprint\runjob.py", line 291, in doit
  File "ptxprint\runjob.py", line 291, in <genexpr>
  File "ptxprint\runjob.py", line 441, in dojob
  File "ptxprint\texmodel.py", line 1129, in convertBook
  File "ptxprint\texmodel.py", line 1074, in flattenModule
  File "ptxprint\usfmutils.py", line 669, in parse
TypeError: unsupported operand type(s) for +: 'int' and 'list'

<class 'TypeError'>: unsupported operand type(s) for +: 'int' and 'list'
```

**Comment by mhosken** [comment]
same as #724

**Comment by mnjames** [comment]
With 2.2.20, I'm now getting the error
```
Traceback (most recent call last):
  File "ptxprint\gtkview.py", line 1253, in onOK
  File "ptxprint", line 223, in doit
  File "ptxprint\runjob.py", line 291, in doit
  File "ptxprint\runjob.py", line 291, in <genexpr>
  File "ptxprint\runjob.py", line 441, in dojob
  File "ptxprint\texmodel.py", line 1129, in convertBook
  File "ptxprint\texmodel.py", line 1074, in flattenModule
  File "ptxprint\usfmutils.py", line 670, in parse
  File "ptxprint\usfmutils.py", line 727, in parse_element
  File "ptxprint\usfmutils.py", line 721, in parse_element
AttributeError: 'Module' object has no attribute 'model'

<class 'AttributeError'>: 'Module' object has no attribute 'model'
```

**Comment by mnjames** [comment]
Not fixed as of 2.2.25. I've sent another archive.

**Comment by mnjames** [comment]
As of 2.2.49 this is still a problem. I've sent another archive demonstrating the error.
